app/views/users/new.html.haml

```diff
@@ -0,0 +1,12 @@
+%section.register.container
+  .row
+    .col-sm-4.col-sm-offset-4
+      = bootstrap_form_for @user, html: { class: "form-horizontal"} do |f|
```

Comment:
If you take a look at the readme for the `rails-bootstrap-forms` gem you will find that horizontal forms are now achieved by using `layout: horizontal`, instead of applying the `form-horizontal` class directly.